### PR TITLE
Added the number of frames as coverage information to record

### PR DESCRIPTION
**Pull request by ggrieco-tob**

### CODE REVIEW

File: lib/Echidna/Exec.hs [hunk 1]
```diff
@@ -104,8 +103,11 @@ execTx = execTxWith vmExcept $ liftSH exec
 type PC = Int
 -- Index per operation in the source code, obtained from the source mapping 
 type OpIx = Int
+-- Stack size from the EVM
+type NumberFrames = Int
```
Comment (incertia) [inline lib/Echidna/Exec.hs:107]: `FrameNumber` maybe?

File: lib/Echidna/Exec.hs [hunk 2]
```diff
@@ -116,16 +118,16 @@ coveragePoints :: CoverageMap -> Int
 coveragePoints = sum . fmap S.size
 
 -- | Given good point coverage, count the number of unique points but
--- only considering the different instruction PCs (discarding the TxResult).
+-- only considering the different instruction PCs (discarding the rest of the fields).
 -- This is useful to report a coverage measure to the user
 scoveragePoints :: CoverageMap -> Int
-scoveragePoints = sum . fmap (S.size . S.map fst3)
+scoveragePoints = sum . fmap (S.size . S.map (\(x, _, _, _) -> x))
```
Comment (incertia) [inline lib/Echidna/Exec.hs:124]: `view _1`

File: lib/Echidna/Output/JSON.hs [hunk 3]
```diff
@@ -23,7 +23,7 @@ data Campaign = Campaign
   , _error :: Maybe String
   , _tests :: [Test]
   , seed :: Int
-  , coverage :: Map String [(PC, OpIx, TxResult)]
+  , coverage :: Map String [ (PC, OpIx, NumberFrames, TxResult)]
```
Comment (incertia) [inline lib/Echidna/Output/JSON.hs:27]: rogue space

**Review by incertia — CHANGES_REQUESTED** [review]

File: lib/Echidna/Exec.hs (hunk 1 above)
Comment (ggrieco-tob) [reply lib/Echidna/Exec.hs:107]: It's the length of the `evm . frames` list.

File: lib/Echidna/Exec.hs (hunk 2 above)
Comment (ggrieco-tob) [reply lib/Echidna/Exec.hs]: Fixed

File: lib/Echidna/Output/JSON.hs (hunk 3 above)
Comment (ggrieco-tob) [reply lib/Echidna/Output/JSON.hs]: Fixed

File: lib/Echidna/Exec.hs (hunk 1 above)
Comment (incertia) [reply lib/Echidna/Exec.hs:107]: `FrameCount` then?

Comment (ggrieco-tob) [reply lib/Echidna/Exec.hs]: Done.

**Pull request merged**